crates/notion-core/src/error/details.rs

```diff
@@ -481,12 +486,15 @@ from {}
 Please verify your internet connection and ensure the correct version is specified.",
                 tool, from_url
             ),
-            ErrorDetails::ExecuteHookError { command } => write!(f, "Could not execute hook command: '{}'
-
-Please ensure that the corrent command is specified.", command),
             ErrorDetails::ExecutablePathError { command } => write!(f, "Could not determine path to executable '{}'
 
 {}", command, REPORT_BUG_CTA),
+            ErrorDetails::ExecutablePermissionsError { bin } => write!(f, "Could not verify permissions for executable '{}'
+
+{}", bin, PERMISSIONS_CTA),
+            ErrorDetails::ExecuteHookError { command } => write!(f, "Could not execute hook command: '{}'
+
+Please ensure that the corrent command is specified.", command),
```

Comment:
Typo! You're looking for `correct` I think?

Reply:
Good catch! Glad I moved that error to put it in the right place alphabetically!